tests/guilds/editGuildMfaLevel.test.ts

```diff
@@ -0,0 +1,12 @@
+import { MfaLevels } from "../../mod.ts";
+import { assertEquals } from "../deps.ts";
+import { loadBot } from "../mod.ts";
+import { CACHED_COMMUNITY_GUILD_ID } from "../utils.ts";
+
+Deno.test("[guild] edit guild mfa level", async () => {
+  const bot = loadBot();
+  await bot.helpers.editGuildMfaLevel(CACHED_COMMUNITY_GUILD_ID, MfaLevels.Elevated, "test");
```

Comment:
CACHED_COMMUNITY_GUILD_ID is not owned by the bot, please make a guild first and use that one then delete that guild in the test